src/frontends/tensorflow/src/utils.cpp

```diff
@@ -32,13 +32,14 @@ void ov::frontend::tensorflow::set_out_name(const std::string& out_name, const o
     output.get_tensor().add_names({out_name});
 }
 
-ov::op::PadType ov::frontend::tensorflow::convert_conv_tf_padding(const ov::frontend::tensorflow::NodeContext& node,
-                                                                  const std::string& tf_padding) {
+ov::op::PadType ov::frontend::tensorflow::convert_tf_padding(const ov::frontend::tensorflow::NodeContext& node,
+                                                             const std::string& tf_padding) {
     auto op_type = node.get_op_type();
 
     TENSORFLOW_OP_VALIDATION(node,
                              op_type == "Conv2D" || op_type == "Conv2DBackpropInput" || op_type == "Conv3D" ||
```

Comment:
it's better to create std::set<std::strings> supported_ops = {"Conv2D", ...};
and check supported_ops.find(op_type) != supported_ops.end()

Reply:
Will be fixed in the next PR. It is not a blocker now to merge since CI is too-too slow.